blocks/hero-block/intl.json

```diff
@@ -0,0 +1,13 @@
+{
+	"hero-block.hello-text": {
+		"de": "Hallo",
+		"en": "Hello World",
+		"es": "Hola Mundo",
+		"fr": "Bonjour le monde",
+		"ja": "こんにちは世界",
+		"ko": "안녕하세요 세계",
+		"no": "Hei Verden",
+		"pt": "Olá Mundo",
+		"sv": "Hej Verden"
```

Comment:
these are test texts. if you don't need these, then they can be removed

Reply:
this has been removed